content/docs/guides/multi-target-exporter.md

```diff
@@ -0,0 +1,492 @@
+---
+title: Understanding and using the multi-target exporter pattern
+---
+
+# Understanding and using the multi-target exporter pattern
+
+This guide will introduce you to the multi-target exporter pattern. To achieve this we will:
+
+* describe the multi-target exporter pattern and why it is used,
+* run the [blackbox](https://github.com/prometheus/blackbox_exporter) exporter as an example of the pattern,
+* configure a custom query module for the blackbox exporter,
+* let the blackbox exporter run basic metric queries against the Prometheus [website](https://prometheus.io),
+* examine a popular pattern of configuring Prometheus to scrape exporters using relabeling.
+
+## The multi-target exporter pattern?
+
+By multi-target [exporter](/docs/instrumenting/exporters/) pattern we refer to a specific design, in which:
+
+* the exporter will get the target’s metrics via a network protocol.
+* the exporter does not have to run on the machine the metrics are taken from.
```

Comment:
(almost) no exporter has to run on the same machine, so this is a bit misleading.

Reply:
Some do. Especially the node-exporter, which is among the first most people come in contact with. This is a guide aimed at people relatively new to Prometheus and I want to make it easy to grasp the concepts while saying the truth.

Reply:
This point is misleading though. That you can run e.g. the haproxy exporter on a different machine doesn't make it multi-target.

Reply:
All of the points have to be true. One is not sufficient. To me that was clear by "in which". But we can change that. Any ideas?

Reply:
It's a bit ambiguous as written. "does not usually" maybe?

Reply:
I am a bit confused. Where exactly would you put the "does not usually"?

Reply:
the exporter does not usually run on the machine